js/data.js

```diff
@@ -0,0 +1,128 @@
+import {getRandomNumberSimple, getRandomNumberFloat,getRandomElement} from './util.js';
+
+const SIMILAR_AD_COUNT = 10;
+
+const TITLES = [
+  'Горячие новинки',
+  'Бесплатная бронь',
+  'Маленькая предоплата',
+  'Без залога',
+  'Возможность продления брони',
+];
+
+const TYPES = [
+  'palace',
+  'flat',
+  'house',
+  'bungalow',
+  'hotel',
+];
+
+const TIMES = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
+
+const FEATURES = [
+  'wifi',
+  'dishwasher',
+  'parking',
+  'washer',
+  'elevator',
+  'conditioner',
+];
+
+const LOCATION_LAT = {
+  min: 35.65000,
+  max: 35.70000,
+  float: 5,
+};
+
+const LOCATION_LNG = {
+  min: 139.70000,
+  max: 139.80000,
+  float: 5,
+};
+
+const PRICES = [
+  100,
+  100000,
+];
+
+const ROOMS = [
+  1,
+  10,
+];
+
+const QUESTS = [
+  1,
+  20,
+];
+
+const DESCRIPTIONS = [
+  'Кондиционер',
+  'Сейф для личных вещей',
+  'Бесплатная парковка',
+  'Не далеко от парка',
+];
+
+const PHOTOS = [
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg',
+];
+
+const avatarNumbers = Array.from({ length: SIMILAR_AD_COUNT }, (v, i) => i + 1);
+
+const avatarRandomNumbers = avatarNumbers.sort(() => Math.random() - 0.5);
+
+const getAvatar = () => {
+  let avatarRandom = avatarRandomNumbers.pop();
+
+  if (avatarRandom < 10) {
+    avatarRandom = `0${String(avatarRandom)}`;
+    return avatarRandom;
```

Comment:
Тут необязательно переназначать переменную и необязательно приводить к строке, можно написать проще
```suggestion
    return `0${avatarRandom}`;
```